clang/lib/DPCT/APINames_ASM.inc

```diff
@@ -67,7 +67,7 @@ ENTRY("dp4a", "dp4a", true, NO_FLAG, P1, "Successful")
 ENTRY("elect", "elect", false, NO_FLAG, P1, "Comment")
 ENTRY("ex2", "ex2", true, NO_FLAG, P1, "Successful")
 ENTRY("exit", "exit", false, NO_FLAG, P1, "Comment")
-ENTRY("fence", "fence", false, NO_FLAG, P1, "Comment")
+ENTRY("fence", "fence", true, NO_FLAG, P1, "Successful")
```

Comment:
I think it should be partial, it's not fully supported.

Reply:
changed